content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,258 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
    Epson and Star receipt printers and Zebra label printers do not need a driver to work. Make sure
    that no driver is selected for those printers.
 
+Epson configuration special case
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Most Epson printers support printing receipts in Odoo :abbr:`POS (Point of Sale)` using the `GS v 0`
+command. However, the following Epson printer models do not support this command:
+
+- TM-U220
+- TM-U230
+- TM-P60
+- TMP-P60II
+
+Bypass this issue by configuring the printer to use the `ESC *` command instead.
+
+Process to force ESC * command
+******************************
+
+Epson printer compatibility
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to check whether the printer is incompatible with `GS v 0` command.
+
+.. seealso::
+   See this `Epson's GS v 0 documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=94>`_ for `GS v 0`
+   compatible printers.
+
+   See this `Epson ESC * documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ for `ESC *`
+   compatible printers.
+
+If the printer is not compatible with the `ESC *` command then the following process is not
+possible. Should the printer be compatible to use the `ESC *` command to print, follow this process
+to configure the printer with the :abbr:`IoT (Internet of Things)` box.
+
+IoT box configuration for ESC *
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To configure the :abbr:`IoT (Internet of Things)` box to use the `ESC *` command to print, go to the
+:abbr:`IoT (Internet of Things)` box homepage. Access the :abbr:`IoT (Internet of Things)` box
+homepage by navigating to :menuselection:`IoT app --> IoT Boxes`. Then click on the :guilabel:`IP
+address` and this will redirect the page to the :abbr:`IoT (Internet of Things)` box homepage.
+
+Choosing the printer
+""""""""""""""""""""
+
+Now click on the :guilabel:`Printers server` button. This will redirect the browser to the *CUPS
+page*. Next, go to :menuselection:`Administration --> Printers --> Add Printer`. Now choose the
+printer that should be modified and click :guilabel:`Continue`.
```

Comment:
```suggestion
Now click on the :guilabel:`Printers server` button. This will redirect the browser to the *CUPS*
page. Next, go to :menuselection:`Administration --> Printers --> Add Printer`, choose the printer
that should be modified, and then click :guilabel:`Continue`.
```